pkg/kubectl/polymorphichelpers/interface.go

```diff
@@ -113,3 +114,9 @@ type RollbackerFunc func(restClientGetter genericclioptions.RESTClientGetter, ma
 
 // RollbackerFn gives a way to easily override the function for unit testing if needed
 var RollbackerFn RollbackerFunc = rollbacker
+
+// ScaleClientFunc gives a way to retrieve a scale getter
+type ScaleClientFunc func(restClientGetter genericclioptions.RESTClientGetter) (scaleclient.ScalesGetter, error)
```

Comment:
Make it a regular helper in cmd/util and we're good.

Reply:
Thanks, done